### PR TITLE
Unnecessary copy removed

### PR DESCRIPTION
**Pull request by KulikovNikita**
Reopened from #2090 by @makart19

### CODE REVIEW

**Comment by KulikovNikita** [comment]
http://intel-ci.intel.com/ed8120f3-bc17-f143-849b-a4bf010d0e2e

**Comment by KulikovNikita** [comment]
[http://intel-ci.intel.com/](http://intel-ci.intel.com/ed8133da-cf30-f1ff-adf0-a4bf010d0e2e)

File: dev/make/cmplr.clang.mk [hunk 1]
```diff
@@ -29,7 +29,7 @@ CORE.SERV.COMPILER.clang = generic
 -DEBC.clang = -g
 
 COMPILER.mac.clang = clang++ -m64 -fgnu-runtime -stdlib=libc++ -mmacosx-version-min=10.14 -fwrapv \
-                     -Werror -Wreturn-type
+                     -Werror -Wreturn-type -falign-functions=16
```
Comment (Alexsandruss) [inline dev/make/cmplr.clang.mk:32]: If this alignment issue is reproduced only locally, then PR might be merged without this change

**Comment by napetrov** [comment]
@KulikovNikita  - why we need alignment options specifically for MacOS?

**Pull request closed without merging**